contractcourt/breach_resolver.go

```diff
@@ -0,0 +1,99 @@
+package contractcourt
+
+import (
+	"encoding/binary"
+	"io"
+
+	"github.com/lightningnetwork/lnd/channeldb"
+)
+
+// breachResolver is a resolver that will handle breached closes. In the
+// future, this will likely take over the duties the current breacharbiter has.
+type breachResolver struct {
+	// resolved reflects if the contract has been fully resolved or not.
```

Comment:
Eventually, we can have this generate the reports we need to populate the `PendingChannels` call.

Reply:
Yeah I omitted that here. I don't think any info like that is persisted currently in the breacharbiter, it's only logged